taxonomy-editor-frontend/src/pages/editentry/AccumulateAllComponents.jsx

```diff
@@ -0,0 +1,52 @@
+import { Button } from "@mui/material";
+import { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import FetchRelations from "./FetchRelations";
+import ListAllEntryProperties from "./ListAllEntryProperties";
+import ListAllOtherProperties from "./ListAllOtherProperties";
+import Swal from 'sweetalert2'
+import withReactContent from 'sweetalert2-react-content'
```

Comment:
what about using [Material ui dialog](https://mui.com/material-ui/api/dialog/)? think it is better to stick with that for styling and so on, also removes the need of an additional library (more things to pass to the client)